src/table/utils/accessibility-utils.ts

```diff
@@ -211,9 +211,9 @@ export const announceSelectionState = (
   }
 };
 
-export const copyCellValue = async (cell: Cell) => {
+export const copyCellValue = async (value?: string) => {
```

Comment:
I found it more practical to use `cell value` instead of cell with cell properties. It makes it much more easier when the function is used for Totals and Head.